packages/core/src/components/Pagination/Pagination.tsx

```diff
@@ -8,6 +9,8 @@ import { ListWrapper, PageNavButton, PageNumberButton } from './Pagination.style
 import PaginationPopup from './PaginationPopup';
 import { PaginationProps } from './types';
 
+const pageNumberStyling = defaultTheme.pagination.pageNumber;
```

Comment:
This will always be medly theme you will not be able to overwrite the theme in your project.